lib/check_request.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Security Scorecard Authors
```

Comment:
pkg is usually for the root package, not for helper stuff. in this case, i dont like the use of lib either as a directory name, what was wrong with using checker itself ? in go, they don't recommend using generic names like utils, base, etc

Reply:
My plan was to have a single package with util fns (I plan to add more in upcoming PRs). But if this is not usually done in  Golang, I'm ok sticking with checker/ itself. Reverted it back to that. PTAL.

Reply:
yes more of a golang thingy - https://blog.golang.org/package-names

Avoid meaningless package names. Packages named util, common, or misc provide clients with no sense of what the package contains. This makes it harder for clients to use the package and makes it harder for maintainers to keep the package focused. Over time, they accumulate dependencies that can make compilation significantly and unnecessarily slower, especially in large programs. And since such package names are generic, they are more likely to collide with other packages imported by client code, forcing clients to invent names to distinguish them.

Break up generic packages. To fix such packages, look for types and functions with common name elements and pull them into their own package.

Reply:
Makes sense.